modules/ROOT/pages/sap/sap-connector-reference-44.adoc

```diff
@@ -0,0 +1,807 @@
+| X.509 Certificate a| String |  Path to the X.509 certificate. This is required when the connector is working as a client. |  | 
+| Application server host a| String |  The host of the SAP application server |  | 
+| Message server configuration a| <<MessageServer>> |  The configuration for message server usage |  | 
+| Enable RFC tracing on server a| Boolean |  Enable or disable RFC trace on the server |  `false` | 
+| Enable CPIC tracing on server a| Enumeration, one of:
+
+** INHERIT
+** NO_TRACING
+** LEVEL_1
+** LEVEL_2
+** LEVEL_3 |  Enable/disable RFC trace on the server. |  | 
+| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  | 
+|===
+[[outbound-config_simple-connection-provider]]
+===== Simple Connection Provider
+
+
+Connection provider that connects using a username and password.
+
+
+====== Parameters
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+| SAP system number a| String |  The two-digit system number (sysnr) of the SAP system. |  | x
+| SAP client ID a| String |  Three-digit client number. Leading zeros are preserved if they appear in the number |  | x
+| Destination Name a| String |  Identifier of the destination generated by this connector. If none is set, then a random value is created. |  | 
+| Login language a| String |  The language code used by the connector. The language determines the code page used for communicating between the SAP Adapter and the application server. |  `EN` | 
+| Extended Properties a| Object |  Use to pass in all the database server-specific connection parameters |  | 
+| SAP username a| String |  The username the connector uses to log in to SAP |  | x
+| SAP password a| String |  The password associated with the login username |  | x
+| Application server host a| String |  The host of the SAP application server |  | 
+| Message server configuration a| <<MessageServer>> |  The configuration for message server usage |  | 
+| Enable RFC tracing on server a| Boolean |  Enable or disable RFC trace on the server | `false` | 
+| Enable CPIC tracing on server a| Enumeration, one of:
+
+** INHERIT
+** NO_TRACING
+** LEVEL_1
+** LEVEL_2
+** LEVEL_3 |  Enable or disable RFC trace on the server |  | 
+| Reconnection a| <<Reconnection>> |  When the application is deployed, a connectivity test is performed on all connectors. If set to true, deployment fails if the test doesn't pass after exhausting the associated reconnection strategy. |  | 
+|===
+
+== Supported Operations
+* <<confirmTransactionId>> 
+* <<createIdoc>> 
+* <<createTransactionId>> 
+* <<executeQueuedRemoteFunctionCall>> 
+* <<executeSynchronousRemoteFunctionCall>> 
+* <<executeTransactionalRemoteFunctionCall>> 
+* <<getFunction>> 
+* <<send>> 
+
+
+
+== Operations
```

Comment:
Move the list of operations from Supported Operations to == Operations and remove the Supported Operations heading.